wpilibNewCommands/src/main/native/cpp/frc2/command/CommandPtr.cpp

```diff
@@ -47,6 +47,19 @@ CommandPtr CommandPtr::Repeatedly() && {
   return std::move(*this);
 }
 
+CommandPtr CommandPtr::Repeatedly(int times) && {
+  AssertValid();
+  std::shared_ptr<int> countPtr = std::make_shared<int>(0);
+  return std::move(*this)
+      .FinallyDo([countPtr = countPtr] { (*countPtr)++; printf("command finished\n");})
+      .Repeatedly()
+      .Until([countPtr = countPtr, times = times] {
+        printf("checking if command should run again... current count: %d, should be: %d, finished:%s\n", *countPtr, times, (*countPtr) >= times ? "true" : "false");
```

Comment:
why print stuff out?

Reply:
it is in the PR's message, that if you were to look at the unit tests, the C++ code doesn't function like the test expects it to do so. This is why I have added these print statements as a separate commit. They will be reverted before the merge